app/liquid_tags/standard_filters.rb

```diff
@@ -1,75 +0,0 @@
-module StandardFilters
```

Comment:
variables are disabled, see test. There's no possibility to use filters anyway:

```ruby
[1] pry(main)> Liquid::Template.parse("{{ a | capitalize }}").render({"a" => "a"})
Liquid::StandardError: Liquid error: Liquid variables are disabled
from devto/lib/liquid/variable.rb:6:in `initialize'
```

also we weren't monkey patching them anyway, as by re-enabling variables and keeping this code filters are still working:

```ruby
[1] pry(main)> Liquid::Template.parse("{{ a | capitalize }}").render({"a" => "a"})
=> "A"
```